suite/tests/client-interface/file_io.dll.c

```diff
@@ -491,3 +494,30 @@ test_times(void)
      */
     dr_get_time(&drtime);
 }
+
+void
+test_vfprintf_helper(file_t f, const char *fmt, ...)
+{
+    va_list ap;
+    ssize_t len1, len2;
+
+    va_start(ap, fmt);
+    len1 = dr_vfprintf(f, fmt, ap);
+    va_end(ap);
+
+    /* check length consistency, because why not. */
+    char buf[100];
+    va_start(ap, fmt);
+    len2 = dr_vsnprintf(buf, 100, fmt, ap);
+    va_end(ap);
+
+    if(len1 != len2 && !(len1 > 100 && len2 == -1)) {
```

Comment:
style: "if ("
(our vera++ style checker, run as part of Travis, should have complained about this...)